scripts.d/support/create_catalog_ng.sh

```diff
@@ -22,8 +22,10 @@ function addToCatalog()
 
     case $name in
         localhost)
+            nbn=$(LANG=C lscpu| grep 'CPU(s):' | sort -u | head -n +1 |awk -F ":" '{print $2}'| tr -d ' ')
             declare -A Catalog=(["name"]=\"$name\"
-                                ["hostname"]=\"$name\")
+                                ["hostname"]=\"$name\"
+                                ["nbOfNodes"]=\"$nbn\")
```

Comment:
Do we really need capital letters here? If not please use current style which is all lower-case letters like hostname.